src/libraries/LibTurnBasedGame.sol

```diff
@@ -46,6 +46,7 @@ library LibTBG {
         bool hasStarted;
         bool hasEnded;
         EnumerableSet.AddressSet players;
+        mapping(address => bool) playerFlags;
```

Comment:
This variable purpose is unclear (name). LibTBG supports storing game specific mechanics via `_getGame`.